indico/modules/events/editing/operations.py

```diff
@@ -460,6 +461,6 @@ def _compose_filepath(editable, revision_file):
     return os.path.join(filepath, filename)
 
 
-def _ensure_publishable_files(editable):
-    if not editable.latest_revision_with_files.has_publishable_files:
+def _ensure_publishable_files(revision):
+    if not revision.editable.has_publishable_files:
```

Comment:
Any idea what the reason for this change was? IMHO it's misleading, because it does NOT check the revision for publishable files, but the latest revision w/ files of the editable:

```py
class Editable(...):
    ...

    @property
    def has_publishable_files(self):
        return self.latest_revision_with_files.has_publishable_files
```

It is in fact the exact same behavior as before, just much less transparent.